utils/b2d.go

```diff
@@ -0,0 +1,62 @@
+package utils
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"path/filepath"
+)
+
+// Get the latest boot2docker release tag name (e.g. "v0.6.0").
+// FIXME: find or create some other way to get the "latest release" of boot2docker since the GitHub API has a pretty low rate limit on API requests
+func GetLatestBoot2DockerReleaseURL() (string, error) {
+	rsp, err := http.Get("https://api.github.com/repos/boot2docker/boot2docker/releases")
+	if err != nil {
+		return "", err
+	}
+	defer rsp.Body.Close()
+
+	var t []struct {
+		TagName string `json:"tag_name"`
+	}
+	if err := json.NewDecoder(rsp.Body).Decode(&t); err != nil {
+		return "", err
+	}
+	if len(t) == 0 {
+		return "", fmt.Errorf("no releases found")
+	}
+
+	tag := t[0].TagName
+	url := fmt.Sprintf("https://github.com/boot2docker/boot2docker/releases/download/%s/boot2docker.iso", tag)
+	return url, nil
+}
+
+// Download boot2docker ISO image for the given tag and save it at dest.
+func DownloadISO(dir, file, url string) error {
```

Comment:
Correct, that should take care of this.  I didn't want to blow this scope too much. :)